packages/theme/src/cli/utilities/asset-ignore.test.ts

```diff
@@ -174,4 +174,65 @@ describe('asset-ignore', () => {
       ])
     })
   })
+
+  describe('applyIgnoreFilters with negated patterns', () => {
+    test(`only negating a file does not produce side effects of other ignored files`, () => {
+      const options = {
+        ignoreFromFile: ['!assets/basic.css'],
+      }
```

Comment:
This test was written because the current behaviour made it so that just having a negated option ignored other files and folders.